weekly.go

```diff
@@ -273,6 +328,14 @@ func createWeeklyReport(title string, value string) {
 	} else {
 		parent := getContentByTitle(space, config.Confluence.WeeklyPath)
 		c = createContent(space, parent.Id, title, value)
+		for _, team := range config.Teams {
+			for _, m := range team.Members {
+				body := bytes.Buffer{}
+				genWeeklyUserPage(&body, m, sprint)
+				userTitle := fmt.Sprintf("%s - %s", m.Name, title)
```

Comment:
For user title, I prefer to just use the end date (i.e. SomeOne - 2019-03-28 instead of SomeOne - 2019-03-22 - 2019-03-28). Because the title suffix here is only used to ensure uniqueness. Shorter form is enough and is more clear.